webapp/src/app/external-modules/material.module.ts

```diff
@@ -45,6 +46,7 @@ const MODULES = [
 	MatTreeModule,
 	MatDividerModule,
 	MatSlideToggleModule,
+	A11yModule,
```

Comment:
Is this used anywhere?

Reply:
Yes, that enables the `cdkFocusInitial` directive